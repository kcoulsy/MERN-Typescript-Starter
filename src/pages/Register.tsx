import React from 'react';

import RegisterForm from '../components/RegisterForm/RegisterForm';

const Register: React.FunctionComponent = () => {
    return (
        <div>
            <h3>Register Page</h3>
            <RegisterForm />
        </div>
    );
};

export default Register;
