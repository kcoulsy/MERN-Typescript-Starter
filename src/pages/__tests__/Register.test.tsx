import React from 'react';
import { shallow } from 'enzyme';

import Register from '../Register';
import { shallowToJson } from 'enzyme-to-json';

describe('render Register component', () => {
    it('should render Register page component', () => {
        const wrapper = shallow(<Register />);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
