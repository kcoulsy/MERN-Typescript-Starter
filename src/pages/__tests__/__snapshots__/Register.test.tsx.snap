// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Register component should render Register page component 1`] = `
<div>
  <h3>
    Register Page
  </h3>
  <Connect(Component) />
</div>
`;
